pkg/skaffold/config/util.go

```diff
@@ -328,6 +330,34 @@ func UpdateGlobalSurveyTaken(configFile string) error {
 	return err
 }
 
+func UpdateGlobalSurveyPrompted(configFile string) error {
+	// Today's date
+	today := current().Format(time.RFC3339)
+	ai := fmt.Sprintf(updateLastPrompted, today)
+	aiErr := fmt.Errorf("could not automatically update the survey prompted timestamp - please run `%s`", ai)
```

Comment:
is this something we actually want the user to do? feels a little weird that we'd tell them to run a command to set a prompt timestamp - maybe we should tell them if this keeps happening to disable instead?

Reply:
if we ask users to disable the prompt, they would never see the prompted.
This command would silence it for 2 weeks. 

How about we add the command to disable permanently here -> https://skaffold.dev/docs/resources/feedback/ or as part of FAQ. 
And change the message to 

```
could not automatically update the survey prompted timestamp - please run skaffold config set --survey --global last-prompted 2019-10-20XXX or 
See <> to permanently disable the prompt. 

```
Or
```

```
could not automatically update the survey prompted timestamp - please run 
skaffold config set --survey --global last-prompted 2019-10-20XXX 
 or  run `skaffold config set --survey --global disable-prompt true` to permanently disable the prompt.
```

Reply:
in any case we're still asking the user to run a command...but after thinking about it a little more I think it's probably fine. we can always change it later if it seems too annoying